docker/Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
             steps {
                 script {
                     LAST_SHA_ROOT = sh (script: "curl -s 'https://api.github.com/repos/the-virtual-brain/tvb-root/commits?sha=tvb-multiscale-cosim' | grep sha | head -1 | tr '\"' '_'", returnStdout: true).trim()
-                    LAST_SHA_MULTISCALE = sh (script: "curl -s 'https://api.github.com/repos/the-virtual-brain/tvb-multiscale/commits' | grep sha | head -1 | tr '\"' '_'", returnStdout: true).trim()
+                    LAST_SHA_MULTISCALE = sh (script: "curl -s 'https://api.github.com/repos/the-virtual-brain/tvb-multiscale/commits?sha=ongoing_dp_work' | grep sha | head -1 | tr '\"' '_'", returnStdout: true).trim()
```

Comment:
This will need to be reverted right before merge into master ?

Reply:
Yes.